### ISSUE TITLE
[Feature] Port ifmatch subtoken for voxel token from EDuke32

### ISSUE THREAD

**Issue by fgsfds** [ZDoom/Raze#1097]
### Which game are you running with Raze?

Duke Nukem 3D

### What Operating System are you using?

Windows 11

### Is your feature request related to a problem? Please describe.

---

### Describe the solution you'd like

Implement ifmatch subtoken for voxel token, so the voxel only replace the metching tile.

https://voidpoint.io/terminx/eduke32/-/commit/30deaa5201b023838a950abbeddc3e0a2833b581

**Comment by coelckers** [comment]
What does this do? Is there any documentation about this feature?

**Comment by fgsfds** [comment]
> What does this do? Is there any documentation about this feature?
I don't think so. It's similar to `tilefromtexture`'s `ifcrc` token, but also includes tile's dimensions and works with voxels.

Examples: 
`voxel "1975.kvx" { ifmatch { size 226 136 crc32 0x6F84404B } tile0 1975 tile1 1979 scale .94 }`
`tilefromtexture 1975 { file "4090.png" xoffset 30 yoffset 107 ifmatch { size 226 136 crc32 0x6F84404B } }`

These entries will only work if the tile 1975 matches  226x136 dimensions and have crc = 0x6F84404B.
